src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -139,6 +139,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         emitAnnotationUpdateEvent: noop,
         getAnnotationsMatchPath: noop,
         getAnnotationsPath: noop,
+        hasNewThreadedReplies: false,
```

Comment:
is the `New` part for this and the other variables needed? is there an oldThreadedReplies?

could we remove `New`?

Reply:
there's `hasReplies` which is the old "threadedReplies".

Reply:
what will the end state look like? is `hasReplies` going anyway? will `hasNewThreadedReplies` be renamed?

Reply:
hasNewThreadedReplies would be going away after GA. `hasReplies` is not a FF, so it will most likely stay there.